foundations/javascript_basics/understanding_errors.md

```diff
@@ -112,41 +118,53 @@ const message = str1.push(str2);
+
+While you should address these warnings if possible and as soon as possible, warnings are not as significant as errors and are more likely to be informational. Warnings are typically shown in yellow, while errors are typically shown in red. Though these colors are not a rule, frequently there will be a visual differentiation between the two, regardless of the platform you are encountering them on.
 
 ### Assignment
 
 <div class="lesson-content__panel" markdown="1">
 
-1. Work through [this lesson on MDN](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/What_went_wrong). Be sure to download their starter code that has intentional errors.
+1. Now, it's time to go through the documentation! Learn more about the [ReferenceError](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ReferenceError), the [SyntaxError](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SyntaxError) and the [TypeError](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypeError) from the MDN Docs.
+1. Go through the [tutorial on the Chrome Debugger](https://developer.chrome.com/docs/devtools/javascript/).
+
+<div class="lesson-note" markdown="1">
+In point 4 of step 3 of the tutorial, devtools will pause on the second line rather than at the declaration of the function. Don't worry this is expected.
+</div>
+
+1. Work through this [lesson on MDN](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/What_went_wrong). Be sure to download their starter code that has intentional errors.
 
 </div>
 
 ### Knowledge check
 
-This section contains questions for you to check your understanding of this lesson on your own. If you’re having trouble answering a question, click it and review the material it links to.
+The following questions are an opportunity to reflect on key topics in this lesson. If you can't answer a question, click on it to review the material, but keep in mind you are not expected to memorize or master this knowledge.
 
 - [What are three reasons why you may see a TypeError?](#type-error)
 - [What is the key difference between an error and a warning?](#errors-vs-warnings)
 - [What is one method you can use to resolve an error?](#tips-for-resolving-errors)
 
 ### Additional resources
 
-This section contains helpful links to other content. It isn't required, so consider it supplemental.
+This section contains helpful links to related content. It isn't required, so consider it supplemental.
 
 - [MDN JavaScript Errors Reference](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors)
+- There are other useful `console` methods like `console.dir` or `console.table`. A [comprehensive list of `console` methods is available at w3schools](https://www.w3schools.com/jsref/obj_console.asp).
```

Comment:
Could you also revert this change to how it was? Again, another change that's part of already-assigned work from #27681.